Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/dialog/CwaDialogHelper.kt

```diff
@@ -0,0 +1,53 @@
+package de.rki.coronawarnapp.ui.dialog
+
+import androidx.fragment.app.Fragment
+
+/**
+ * Helper function that creates a CwaDialog and displays it.
+ * Usage:
+ *
+ * ```kotlin
+ * createDialog {
+ *     title { "Title example" }
+ *     message { "Message example" }
+ *     positiveButton("Positive button text") { action { foo1() }
+ *     negativeButton("Negative button text") { action { foo2() }
+ *     neutralButton("Neutral button text") { action { foo3() }
```

Comment:
I think the doc here needs an update ?

Reply:
Yes id tdoes

Reply:
`message` and `title` don't require lambdas too